invenio_rdm_records/services/components.py

```diff
@@ -174,16 +175,28 @@ def _validate_pids(self, pids):
                 field_name="pids",
             )
 
-    def create(self, identity, data=None, record=None, **kwargs):
+    def _remove_invalid_pids(self, pids, errors):
+        """Remove pids that have validation errors."""
+        errors = errors or []
+        for error in errors:
+            if "pids._schema" == error["field"]:
+                for message in error["messages"]:
+                    # assume format "[some text] scheme {scheme}"
+                    pid_type = message.split("scheme")[1].strip()
+                    pids.pop(pid_type, None)
+
+    def create(self, identity, data=None, record=None,  errors=None):
```

Comment:
Did the service interface change or we did a mistake in copying it? I don't recall this `errors` param :(